JS/Lesson18/esra/script.js

```diff
@@ -136,14 +166,41 @@ console.log('----------------------------');
 // Output: {name: "John", age: 30}
 const testObjEx11 = {name: "John", age: 30, city: "New York"};
 
+function removeObjectInTheKey(objects, keys) {
```

Comment:
The name of the function is a bit confusing right now. I'd say `removeFromObject(object, keyToRemove)` would be a better choice